crmprtd/crmprtd_log.txt

```diff
@@ -0,0 +1,3967 @@
+2018-05-25 14:42:12,310:DEBUG:crmprtd.wamr - Processing observations
```

Comment:
``crmprtd_log.txt`` should not be part of this (or any) commit.

Reply:
You are right, I'm not sure how that got in there. I'll take it out.